slam-libraries/viam-cartographer/README.md

```diff
@@ -19,6 +19,16 @@ Learn more at [the official Docs site](https://google-cartographer.readthedocs.i
 ## Installation instructions
 Make sure to follow all steps as outlined in [the setup section here](../../README.md#setup).
 
+### Automatic Dependency Installation (x64, arm64, or macOS)
```

Comment:
this section tells users they can automatically install dependencies if they want

Reply:
Right! ok makes sense